### PR TITLE
add Chinese translation

### PR DESCRIPTION
**Pull request by sbc-trtl**
I made a Chinese translation. see if no problem

### CODE REVIEW

**Comment by SoreGums** [comment]
ha, too slow @ar-x  :P

hi @sbc-trtl please update the urls to be https:// instead of http:// - thanks so much!!

**Comment by sbc-trtl** [comment]
ok, i would

File: _config.yml [hunk 1]
```diff
@@ -16,13 +16,16 @@ gems:
 
 
 # Localization config
-languages: ["en", "af", "fr", "it"]
+languages: ["en", "af", "fr", "it", "cn"]
```
Comment (ar-x) [inline _config.yml]: Is this Mandarin, Simplified Chinese?

Comment (ar-x) [reply _config.yml]: I believe it should be `zh` instead of `cn`, here and below (including the file name).

Comment (sbc-trtl) [reply _config.yml]: Mandarin

Comment (sbc-trtl) [reply _config.yml]: thx, i should change it to zh-tw

Comment (ar-x) [reply _config.yml]: @sbc-trtl For my own edification, is the simplified chinese or traditional chinese?

File: _config.yml [hunk 2]
```diff
@@ -16,13 +16,16 @@ gems:
...
+languages: ["en", "af", "fr", "it", "zh"]
 #languages: ["en", "af", "fr", "it", "dev"] # enable during development to see text keys at /dev/
 
 language_names:
   af:
     short: af
     full: Afrikaans
+  zh:
+    short: zh
+    full: Mandarin Chinese
```
Comment (ar-x) [inline _config.yml]: @sbc-trtl This text should be in Chinese – like `中文` (whatever is appropriate) - it gets shown on the top right language selection dropdown.

File: _i18n/zh.yml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  story_4: 事實上，大多數的烏龜幣開發人員都有其他的全職工作，而烏龜幣是提供我們樂趣並回饋給社群的一個方式。
+  story_5: 烏龜幣是如何產生的?
+  story_6: 在12月、一個寒冷的夜晚，駭客 RockSteady 及 Bebop 兩人喝了太多啤酒，並且討論到世界上有太多造假、而實際上沒有產品開發實力的區塊鍊項目ICO。Bebop 認為在這個時代，若沒有認真的團隊、認真的行銷、以及正式的經費來源，不可能建立一個有用的區塊鍊項目。
+  story_7: RockSteady 熱切地摸著鬍子，仔細地聆聽 Bebop。他只有一個反應..
+  story_8: <strong>"拿好我的啤酒，我做給你看"</strong>
+  story_9: 這個夜晚，2017年12月9日，烏龜幣誕生了。
+  story_10: 為何創造烏龜幣?
+  story_11: 許多的區塊鍊項目獲得了 <strong>太多的經費支持</strong> 而大多數 <strong>根本沒有實際產出</strong>。我們理解到一點 "為何不讓整個事情倒著來?" 發起一個好玩的區塊鍊項目，讓它儘可能的容易使用且容易獲得、讓它自己成長。
+  story_12: 我們強烈地相信我們的無經費策略；如果我們做出好的產品， <strong>產品會自己創造價值</strong>。 如果你認為我們是錯的，我們歡迎您來 <a href="http://meta.turtlecoin.lol">談談</a> 你的想法。
+  story_13: 從前有一隻聰明的老鼠說過: "烏龜只有伸出脖子才能向前走.."
```
Comment (ar-x) [inline _i18n/zh.yml]: YAML is colon `:` sensitive. Do you'll need to quote the entire string.

```
"從前有一隻聰明的老鼠說過: \"烏龜只有伸出脖子才能向前走..\""
```

**Comment by ar-x** [comment]
@sbc-trtl You mentioned in Discord that you got someone to review this PR. But I don't see any comments, or a review approval. Did I miss something?

**Pull request merged**